redhat-director-scripts/rhosp16.1/puppet/trilio/manifests/horizon.pp

```diff
@@ -1,15 +1,34 @@
 class trilio::horizon (
     $horizon_dir                        = '/etc/openstack-dashboard',
+    $barbican_api_enabled               = hiera('barbican_api_enabled'), 
 ){
 
   tag 'triliohorizonconfig'
 
-    file { "${horizon_dir}/local_settings.d":
-        ensure => 'directory',
-    }->
-    file { "${horizon_dir}/local_settings.d/_002_trilio_dashboard.py":
-        ensure  => present,
-        content => template('trilio/_002_trilio_dashboard.py.erb'),
-    }   
+   if $barbican_api_enabled {
+       file_line { "ENABLE OPENSTACK_ENCRYPTION_SUPPORT":
+           ensure  => present,
+           path   => "${horizon_dir}/local_settings",
+           line   => 'OPENSTACK_ENCRYPTION_SUPPORT = True',
+           match  => '^OPENSTACK_ENCRYPTION_SUPPORT',
+       }
+   }
+   else {
+       file_line { "DISABLE OPENSTACK_ENCRYPTION_SUPPORT":
+           ensure  => present,
+           path   => "${horizon_dir}/local_settings",
+           line   => 'OPENSTACK_ENCRYPTION_SUPPORT = False',
+           match  => '^OPENSTACK_ENCRYPTION_SUPPORT',
+       }
+   }
 
-}
\ No newline at end of file
+
+    file_line { "ENABLE TRILIO_ENCRYPTION_SUPPORT":
+        ensure => present,
+        path   => "${horizon_dir}/local_settings",
+        line   => 'TRILIO_ENCRYPTION_SUPPORT = True',
```

Comment:
when the variable set to False?

Reply:
barbican_api_enabled get's false in that case  OPENSTACK_ENCRYPTION_SUPPORT value set to the False

Reply:
I don't see the False setting code in `if barbican_api_enabled` block.
and the TRILIO_ENCRYPTION_SUPPORT = True is also not depend on any condition.
Am I missing anything here?